### DISCUSSION TITLE
Will scratch addons soon still be a extension, or not?

### DISCUSSION THREAD

**Discussion by WojtekCodesToday** [ScratchAddons/ScratchAddons#4283] [Help]
Will the extension in the future be something different that still works?

**Comment by WorldLanguages** [comment]
What do you mean by "something different"?

**Comment by WojtekCodesToday** [comment]
> What do you mean by "something different"?

I mean something else that isn't a extension.

**Reply by rimopa** [reply]
Like a desktop app with Scratch implemented?

**Reply by WojtekCodesToday** [reply]
Maybe

**Reply by Dhuls-Scratch** [reply]
> Like a desktop app with Scratch implemented?

That already exists. It's called [TurboWarp Desktop](https://desktop.turbowarp.org/).

**Reply by WojtekCodesToday** [reply]
> > Like a desktop app with Scratch implemented?
> 
> That already exists. It's called [TurboWarp Desktop](https://desktop.turbowarp.org/).

this is called online version, and yes that's a desktop app

**Reply by CST1229** [reply]
> > > Like a desktop app with Scratch implemented?
> > 
> > 
> > That already exists. It's called [TurboWarp Desktop](https://desktop.turbowarp.org/).
> 
> this is called online version, and yes that's a desktop app

Online Scratch with SA is still [TurboWarp](https://turbowarp.org).
